File: backend/tests/fixtures/parsers/kotlin/hello.kt
package com.example

import kotlin.text.uppercase

class Greeter : Base() {
    fun greet(name: String) {
        val formatted = formatName(name)
        println(formatted)
    }

    fun formatName(name: String): String {
        return name.uppercase()
    }
}

fun standaloneHelper() {
    Greeter().greet("world")
}

File: backend/tests/fixtures/parsers/ruby/hello.rb
require 'json'

class Greeter < Base
  def greet(name)
    formatted = format_name(name)
    puts(formatted)
  end

  def format_name(name)
    name.upcase
  end
end

def standalone_helper
  Greeter.new.greet("world")
end
